### DISCUSSION TITLE
EGit and Hooks

### DISCUSSION THREAD

**Discussion by LeGrosWinnie** [eclipse-egit/egit#70] [Q&A]
Hello,

Question asked many times but I still have issue with hooks.
I managed to get them to be executed (add git for windows path to User PATH) but I cannot read the message.
Instead of having a console or terminal output it is a small empty popup :
![Hook Result](https://github.com/user-attachments/assets/fcaee12c-6b4f-4006-a36e-a31d28ab1acd)

Any idea ?

Thanks.

**Comment by Bananeweizen** [comment]
The message is created from the _error_ output of the hook script, not from the _standard_ output, see https://github.com/eclipse-egit/egit/blob/11d184c1118d3574c4b8026de482df1e7d4e6431/org.eclipse.egit.ui/src/org/eclipse/egit/ui/internal/commit/CommitJob.java#L195-L197. I guess your script doesn't write the error to stderr.
